src/dinit.cc

```diff
@@ -365,6 +375,11 @@ static int process_commandline_arg(char **argv, int argc, int &i, options &opts)
             }
         }
         #endif
+        #ifdef SUPPORT_SELINUX
+        else if (strcmp(argv[i], "--disable-selinux") == 0) {
```

Comment:
This option name is misleading. Could we name it something more descriptive of what it actually does? "--disable-selinux-policy" would be a bit better.